src/main/java/bio/terra/cli/command/Status.java

```diff
@@ -1,29 +1,64 @@
 package bio.terra.cli.command;
 
-import bio.terra.cli.context.GlobalContext;
-import bio.terra.cli.context.WorkspaceContext;
-import java.util.concurrent.Callable;
+import bio.terra.cli.command.helperclasses.CommandSetup;
+import bio.terra.cli.command.helperclasses.FormatFlag;
+import bio.terra.cli.context.ServerSpecification;
+import bio.terra.workspace.model.WorkspaceDescription;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the second-level "terra status" command. */
 @Command(name = "status", description = "Print details about the current workspace.")
-public class Status implements Callable<Integer> {
+public class Status extends CommandSetup {
 
+  @CommandLine.Mixin FormatFlag formatFlag;
+
+  /** Build the return value from the global and workspace context. */
   @Override
-  public Integer call() {
-    GlobalContext globalContext = GlobalContext.readFromFile();
-    WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
+  protected void execute() {
+    StatusReturnValue statusReturnValue =
+        new StatusReturnValue(globalContext.server, workspaceContext.terraWorkspaceModel);
+    formatFlag.printReturnValue(statusReturnValue, returnValue -> this.printText(returnValue));
+  }
+
+  /** POJO class for printing out this command's output. */
+  public static class StatusReturnValue {
```

Comment:
do these "return value" classes still need to be `public`?
optional: do we have any more preferable naming terminology than "return value"? I don't hate it, but the values aren't being returned anymore so much as printed or outputted? brainstorming: `StatusOutput`, `StatusOutputValue`, `StatusPrintValue`? not sure those are better, up to you

Reply:
Good catch. Changed "return value" classes to `private`.

RE: "return value" & other options. I think "output" or "print" would be slightly misleading for commands that output things besides the end object (e.g. docker commands that stream output, print progress messages). We don't have any of those commands yet, but I think they'll come in the future. I'm open to other suggestions, but I think "return" is slightly clearer for those cases than "print" or "output".